classroom-template-bundle-spec.md

```diff
@@ -0,0 +1,168 @@
+
+on_demand_url:
+  locales:
+    en: "https://www.coursera.org/specializations/architecting-hybrid-cloud-infrastructure-anthos"
+
+tags: [sample, life-changing, gcp]
+product_tags: ['compute engine', 'cloud storage']
+role_tags: ['cloud architect', 'developers backend']
+domain_tags: ['infrastructure']
+level: 1
+
+# Allow classroom scheduling for this ClassroomTemplate
+published: true
+
+# Estimated time to take the course, in days
+estimated_duration_days: 1
+
+# Estimated time to take the course, in minutes
+estimated_duration: 60
+
+# Allow lab order to be changed by the trainer
+lock_position: false
+
+# Trainers must stay within the limits
+max_hot_labs: 30
+
+# Enable DRM on instructions if supported
+enable_drm: false
+
+# Check classroom for fraud
+resource_limit_check: false
+
+# Instructions that will be surfaced to students
+student_resources: ...
+
+# Instructions that will not be surfaced to students, but may be referenced by an instructor
+instructor_resources: ...
+
+# The important part of a ClassroomTemplate which lists all of the labs
+modules: ...
+
+```
+
+Note that all of the main chunks of localized content (title, description, objectives, audience, prerequisites, etc.) are HTML content that may be displayed in various contexts. All of these chunks will be sanitized according to the restricted set in the [HTML spec](./html-spec.md).
+
+### Attribute specification
+The full specification is as follows:
+
+attribute               | required | type        | notes
+----------------------- | -------- | ----------- | -----------------------------------------
+entity_type             | ✓        | string      | Must be `ClassroomTemplate`
+schema_version          | ✓        | integer     |
+default_locale          | ✓        | enum        | Must be a valid locale code
+version                 |          | string      | Version string used as content decorator
+classroom_type          |          | enum        | One of ["Self-paced", "Bootcamp/Workshop", "Instructor-led"]
+course_code             |          | string      | Id for Course, e.g. T_AHYXXX-I
+title                   | ✓        | dictionary  | A locale dictionary of titles
+description             | ✓        | dictionary  | A locale dictionary of descriptions
+objectives              |          | dictionary  | A locale dictionary of objectives
+audience                |          | dictionary  | A locale dictionary of audiences
+prerequisites           |          | dictionary  | A locale dictionary of prerequisites
+outline                 |          | dictionary  | A locale dictionary of outline elements usually formatted as a JSON string
+on_demand_url           |          | dictionary  | A locale dictionary of URL's to Coursera (or other) on-demand content
```

Comment:
- "on-demand" doesn't mean the same thing to everyone, and we're about to start delivering "on-demand" courses on Qwiklabs proper.
- There's more than one of them.

How about `external_delivery_urls`? (I'm not sold on this either)

Reply:
Maybe `external_content_url` ?